amoro-ams/src/main/java/org/apache/amoro/server/dashboard/controller/LoginController.java

```diff
@@ -49,10 +50,11 @@ public void login(Context ctx) {
     Map<String, String> bodyParams = ctx.bodyAsClass(Map.class);
     String user = bodyParams.get("user");
     String pwd = bodyParams.get("password");
-    if (adminUser.equals(user) && (adminPassword.equals(pwd))) {
-      ctx.sessionAttribute("user", new SessionInfo(adminUser, System.currentTimeMillis() + ""));
-      ctx.json(OkResponse.of("success"));
```

Comment:
Seems this part of logic is missed after `dashboardServer.getLoginAuthProvider().authenticate(credential);`